mappings/net/minecraft/world/attribute/AttributeValidator.mapping

```diff
@@ -0,0 +1,11 @@
+CLASS net/minecraft/class_12191 net/minecraft/world/attribute/AttributeValidator
```

Comment:
Note how this class is generic - there is nothing that stops people from using this for non-numeric values. Also note that `Range` conflicts with another class.